modules/ROOT/pages/mock-file-cookbook.adoc

```diff
@@ -0,0 +1,68 @@
+= Mocking Resources for Your Tests
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+As your tests suites start to get bigger and more complex, it’s a good practice to externalize some parts of the test code, not only so that you can keep the code clean and organized but also so that you can reuse the files that you need to mock for your tests.
```

Comment:
I feel too many "so that you can" what do you think @fermujica

Reply:
Agreed.